problems/objects.js

```diff
@@ -187,15 +282,42 @@ const countOccurance = () => {};
  * @returns {Object} Counts all characters except spaces  {a: 2, g: 1, o: 2, d:1, s: 1, n:a, k:1, e: 1}
  */
 
-const countOccuranceNoSpaces = () => {};
+const countOccuranceNoSpaces = (str) => {
+  let count = {}
+  let newStr = str.split(" ").join("")
```

Comment:
Instead of spliting then joining, use 
if(count[lowerCase] === " ") {
  continue;
}